README.md

```diff
@@ -0,0 +1,144 @@
+
+**Hello World Webpack App:** This project employs a Ruby on Rails app with PostgreSQL database, react and redux to practice creating one application using for both front end and back end.This is rendering random greeting to the user. 📚
+
+## 🛠 Built With <a name="built-with"></a>
+
+  <ul>
+    <li>React</li>
+    <li>Redux</li>
+    <li>Ruby</li>
+    <li>Ruby On Rails</li>
+    <li>PostgreSQL</li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Display Random greeting**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+  * A basic understanding of Ruby, Rails and Git
+  * A postgresql connection
+  * Create some data before running to see Greatings or run rails db:seed
+### Usage
+1. Install Ruby
+1. Install Rails
+2. Clone this repo 
+
+```
+git clone git@github.com:HafizMassamTabraizKhan/hello-rails-react.git
+```
+
+To run the project, execute the following command:
+
+```
+  rails s
+```
+
+Or
+
+```
+   ./bin/dev
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+### 👤 **HafizMassamTabraizKhan**
+
+- GitHub: [@HafizMassamTabraizKhan](https://github.com/HafizMassamTabraizKhan)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/HafizMassamTabraizKhan)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Modern UI**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please support this project, your contribution can make a significant difference and help us achieve our goals better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Grateful to all for inspiring codebase. Thank you!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## FAQ <a name="faq"></a>
+
+- **Can I fork and reuse this repository?**
+
+Absolutely! Please feel free to fork and reuse this repository for your own projects.
+
+- **Is it okay to improve this repository? Will my changes be accepted?**
+
+Definitely! We welcome any improvements or new ideas you may have. If your changes are good and align with the project's goals, we would be happy to accept them.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] @HafizMassamTabraizKhan Great work on your readme however the license link is broken, kindly consider how you are referencing the license file.